webserver/web/templates/paste/index.html.tera

```diff
@@ -46,6 +46,9 @@
 {%- endblock head %}
 
 {% block title -%}
+  {%- if paste.author and paste.author.name -%}
+{{ paste.author.name }} authored
```

Comment:
I'm concerned this might need a trailing space in some rare browsers. But it works on major ones:
https://liefland.net/~zarthus/pastegg.html

Reply:
I'm assuming `{{ ... }}` escapes by default?

Reply:
`paste.author.username` might be preferable 🤷‍♀️